### DISCUSSION TITLE
Object storage (S3 compatible) for farming

### DISCUSSION THREAD

**Discussion by musicjeremy** [Chia-Network/chia-blockchain#3433] [Ideas]
Currently, I am successfully farming with my plots stored in an S3 compatible object store. To do this, I have to mount the object store as a FUSE filesystem. It works, but it's a layer of complexity.

Is there any plan to make the farmer better aware of object storage?

Can I help?

**Comment by yapwenbin** [comment]
Any latency issue?

**Reply by musicjeremy** [reply]
I'm well within the 30 second limit, but always looking to do better.
